assets/js/renderer.js

```
@@ -1,186 +1,179 @@
 // Actions
 const generateKeys = async () => {
-    keyType = document.getElementById('select-keyType').value
-    let passphrase = document.getElementById('passphrase-input').value || undefined;
-    let { privateKey, publicKey } = await window.utils.generateKeys(keyType, passphrase)
-    document.getElementById('private-key-text-area').value = privateKey
-    document.getElementById('public-key-text-area').value = publicKey
-
-    //Enable buttons
-    change_buttons_disabled_state(false)
-
+  keyType = document.getElementById('select-keyType').value
```

ESLint

[error] 'keyType' is not defined.

(no-undef)